test/mint.js

```diff
@@ -318,4 +318,41 @@ describe('PodCastNFT', function () {
       )
     })
   })
+
+  describe('burn', () => {
+    it('can burn for admin', async () => {
+      const mintTx = await contract.mint(
+          'https://example.com/podcast.png',
+          ['Podcast Contributor'],
+          '10000',
+          owner.address
+      )
+      await mintTx.wait()
+
+      const transferTx = await contract['safeTransferFrom(address,address,uint256)'](
```

Comment:
as far as I understand, we don't need to transfer before burn. burn is deleting state

Reply:
fixed in https://github.com/henkaku-center/podcast-nft/pull/33/commits/93adf89a6766a8579746ff6c324c62357fd82e5d